mojaloop-simulator/values.yaml

```diff
@@ -490,7 +490,7 @@ defaults: &defaults
         # The name of the sqlite log file. This probably doesn't matter much to the user, except that
         # setting :memory: will use an in-memory sqlite db, which will be faster and not consume disk
         # space. However, it will also mean that the logs will be lost once the container is stopped.
-        SQLITE_LOG_FILE: ./log.sqlite
+        SQLITE_LOG_FILE: ':memory:'
```

Comment:
`./log.sqlite` - before the upgrade, sqlite seemed fine to accept a file that doesn't exist, but now will throw a error if file is missing. It should be fine to use memory here as well.